packages/renderer/src/lib/preferences/PreferencesCliToolsRendering.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+import { cliToolInfos } from '../../stores/cli-tools';
+import Markdown from '../markdown/Markdown.svelte';
+import SettingsPage from './SettingsPage.svelte';
+import EngineIcon from '../ui/EngineIcon.svelte';
+import EmptyScreen from '../ui/EmptyScreen.svelte';
+</script>
+
+<SettingsPage title="CLI Tools">
+  <div class="h-full">
+    <EmptyScreen
+      aria-label="no-resource-panel"
+      icon="{EngineIcon}"
+      title="No CLI Tool description found"
+      message="Start an extension that add CLI Tool description"
+      hidden="{$cliToolInfos.length > 0}" />
+
+    {#each $cliToolInfos as cliTool}
+      <div class="bg-charcoal-600 mb-5 rounded-md p-3 divide-x divide-gray-900 flex">
+        <div>
+          <!-- left col - cli-tool icon/name + "create new" button -->
+          <div class="min-w-[170px] max-w-[200px]">
+            <div class="flex">
+              {#if cliTool?.images?.icon}
+                {#if typeof cliTool.images.icon === 'string'}
+                  <img src="{cliTool.images.icon}" alt="{cliTool.name}" class="max-w-[40px] h-full" />
+                {/if}
+              {/if}
+              <span class="my-auto text-gray-400 ml-3 break-words">{cliTool.name}</span>
+            </div>
+          </div>
+        </div>
+        <!-- cli-tools columns -->
+        <div class="grow flex-column divide-gray-900 ml-2">
+          <span class="my-auto text-gray-400 ml-3 break-words">{cliTool.displayName}</span><span
+            class="float-right text-sm">Provided by {cliTool.providedBy}</span>
```

Comment:
```suggestion
            class="float-right text-sm">Registered by {cliTool.providedBy}</span>
```

I think provided is misleading as for example I can have kubectl binary being installed by myself but if an extension find it installed globally it would say that it's provided by this extension while it's more "registered" (but not for example managed or provided)